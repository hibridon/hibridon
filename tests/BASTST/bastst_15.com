inp=bastst_15.inp
out=bastst_15.out
run
save=save_15.inp