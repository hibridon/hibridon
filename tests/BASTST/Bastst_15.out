      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** 2P-DIAT system parameters:
 NTERM  =   1        IOP    =   1        JMAX   =  10
 BROT   = 0.8000     ASO    = 2.0000E-02
 LAMMIN:    1
 LAMMAX:   10
 MPROJ:     0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= T
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: -2; JOUT:  0   2
 INDOUT:    0   1
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-24 at 10:41
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 20ca540af55384f2ca166f283bde32c5da56d99e
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 24 11:02:02 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISOTROPIC TERMS=10

 **  CC MOL + 2P ATOM ** RMU=  16.4700       E=  5000.00   JTOT=   20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N   JA(IS)  JMOL  K(J)    L    CENT    EINT(CM-1)
   1     0.5     0     0    21   462        -0.020
   2     1.5     1     0    19   380         0.010
   3     1.5     1     0    21   462         0.010
   4     0.5     0     1    21   462         1.580
   5     0.5     1     1    19   380         1.580
   6     0.5     1     1    21   462         1.580
   7     1.5     0     1    21   462         1.610
   8     1.5     1     1    19   380         1.610
   9     1.5     1     1    21   462         1.610
  10     1.5     2     1    19   380         1.610
  11     1.5     2     1    21   462         1.610
  12     1.5     2     1    23   552         1.610
  13     0.5     1     2    19   380         4.780
  14     0.5     1     2    21   462         4.780
  15     0.5     2     2    19   380         4.780
  16     0.5     2     2    21   462         4.780
  17     0.5     2     2    23   552         4.780
  18     1.5     0     2    21   462         4.810
  19     1.5     1     2    19   380         4.810
  20     1.5     1     2    21   462         4.810
  21     1.5     2     2    19   380         4.810
  22     1.5     2     2    21   462         4.810
  23     1.5     2     2    23   552         4.810
  24     1.5     3     2    17   306         4.810
  25     1.5     3     2    19   380         4.810
  26     1.5     3     2    21   462         4.810
  27     1.5     3     2    23   552         4.810
  28     0.5     2     3    19   380         9.580
  29     0.5     2     3    21   462         9.580
  30     0.5     2     3    23   552         9.580
  31     0.5     3     3    17   306         9.580
  32     0.5     3     3    19   380         9.580
  33     0.5     3     3    21   462         9.580
  34     0.5     3     3    23   552         9.580
  35     1.5     1     3    19   380         9.610
  36     1.5     1     3    21   462         9.610
  37     1.5     2     3    19   380         9.610
  38     1.5     2     3    21   462         9.610
  39     1.5     2     3    23   552         9.610
  40     1.5     3     3    17   306         9.610
  41     1.5     3     3    19   380         9.610
  42     1.5     3     3    21   462         9.610
  43     1.5     3     3    23   552         9.610
  44     1.5     4     3    17   306         9.610
  45     1.5     4     3    19   380         9.610
  46     1.5     4     3    21   462         9.610
  47     1.5     4     3    23   552         9.610
  48     1.5     4     3    25   650         9.610
  49     0.5     3     4    17   306        15.980
  50     0.5     3     4    19   380        15.980
  51     0.5     3     4    21   462        15.980
  52     0.5     3     4    23   552        15.980
  53     0.5     4     4    17   306        15.980
  54     0.5     4     4    19   380        15.980
  55     0.5     4     4    21   462        15.980
  56     0.5     4     4    23   552        15.980
  57     0.5     4     4    25   650        15.980
  58     1.5     2     4    19   380        16.010
  59     1.5     2     4    21   462        16.010
  60     1.5     2     4    23   552        16.010
  61     1.5     3     4    17   306        16.010
  62     1.5     3     4    19   380        16.010
  63     1.5     3     4    21   462        16.010
  64     1.5     3     4    23   552        16.010
  65     1.5     4     4    17   306        16.010
  66     1.5     4     4    19   380        16.010
  67     1.5     4     4    21   462        16.010
  68     1.5     4     4    23   552        16.010
  69     1.5     4     4    25   650        16.010
  70     1.5     5     4    15   240        16.010
  71     1.5     5     4    17   306        16.010
  72     1.5     5     4    19   380        16.010
  73     1.5     5     4    21   462        16.010
  74     1.5     5     4    23   552        16.010
  75     1.5     5     4    25   650        16.010
  76     0.5     4     5    17   306        23.980
  77     0.5     4     5    19   380        23.980
  78     0.5     4     5    21   462        23.980
  79     0.5     4     5    23   552        23.980
  80     0.5     4     5    25   650        23.980
  81     0.5     5     5    15   240        23.980
  82     0.5     5     5    17   306        23.980
  83     0.5     5     5    19   380        23.980
  84     0.5     5     5    21   462        23.980
  85     0.5     5     5    23   552        23.980
  86     0.5     5     5    25   650        23.980
  87     1.5     3     5    17   306        24.010
  88     1.5     3     5    19   380        24.010
  89     1.5     3     5    21   462        24.010
  90     1.5     3     5    23   552        24.010
  91     1.5     4     5    17   306        24.010
  92     1.5     4     5    19   380        24.010
  93     1.5     4     5    21   462        24.010
  94     1.5     4     5    23   552        24.010
  95     1.5     4     5    25   650        24.010
  96     1.5     5     5    15   240        24.010
  97     1.5     5     5    17   306        24.010
  98     1.5     5     5    19   380        24.010
  99     1.5     5     5    21   462        24.010
 100     1.5     5     5    23   552        24.010
 101     1.5     5     5    25   650        24.010
 102     1.5     6     5    15   240        24.010
 103     1.5     6     5    17   306        24.010
 104     1.5     6     5    19   380        24.010
 105     1.5     6     5    21   462        24.010
 106     1.5     6     5    23   552        24.010
 107     1.5     6     5    25   650        24.010
 108     1.5     6     5    27   756        24.010
 109     0.5     5     6    15   240        33.580
 110     0.5     5     6    17   306        33.580
 111     0.5     5     6    19   380        33.580
 112     0.5     5     6    21   462        33.580
 113     0.5     5     6    23   552        33.580
 114     0.5     5     6    25   650        33.580
 115     0.5     6     6    15   240        33.580
 116     0.5     6     6    17   306        33.580
 117     0.5     6     6    19   380        33.580
 118     0.5     6     6    21   462        33.580
 119     0.5     6     6    23   552        33.580
 120     0.5     6     6    25   650        33.580
 121     0.5     6     6    27   756        33.580
 122     1.5     4     6    17   306        33.610
 123     1.5     4     6    19   380        33.610
 124     1.5     4     6    21   462        33.610
 125     1.5     4     6    23   552        33.610
 126     1.5     4     6    25   650        33.610
 127     1.5     5     6    15   240        33.610
 128     1.5     5     6    17   306        33.610
 129     1.5     5     6    19   380        33.610
 130     1.5     5     6    21   462        33.610
 131     1.5     5     6    23   552        33.610
 132     1.5     5     6    25   650        33.610
 133     1.5     6     6    15   240        33.610
 134     1.5     6     6    17   306        33.610
 135     1.5     6     6    19   380        33.610
 136     1.5     6     6    21   462        33.610
 137     1.5     6     6    23   552        33.610
 138     1.5     6     6    25   650        33.610
 139     1.5     6     6    27   756        33.610
 140     1.5     7     6    13   182        33.610
 141     1.5     7     6    15   240        33.610
 142     1.5     7     6    17   306        33.610
 143     1.5     7     6    19   380        33.610
 144     1.5     7     6    21   462        33.610
 145     1.5     7     6    23   552        33.610
 146     1.5     7     6    25   650        33.610
 147     1.5     7     6    27   756        33.610
 148     0.5     6     7    15   240        44.780
 149     0.5     6     7    17   306        44.780
 150     0.5     6     7    19   380        44.780
 151     0.5     6     7    21   462        44.780
 152     0.5     6     7    23   552        44.780
 153     0.5     6     7    25   650        44.780
 154     0.5     6     7    27   756        44.780
 155     0.5     7     7    13   182        44.780
 156     0.5     7     7    15   240        44.780
 157     0.5     7     7    17   306        44.780
 158     0.5     7     7    19   380        44.780
 159     0.5     7     7    21   462        44.780
 160     0.5     7     7    23   552        44.780
 161     0.5     7     7    25   650        44.780
 162     0.5     7     7    27   756        44.780
 163     1.5     5     7    15   240        44.810
 164     1.5     5     7    17   306        44.810
 165     1.5     5     7    19   380        44.810
 166     1.5     5     7    21   462        44.810
 167     1.5     5     7    23   552        44.810
 168     1.5     5     7    25   650        44.810
 169     1.5     6     7    15   240        44.810
 170     1.5     6     7    17   306        44.810
 171     1.5     6     7    19   380        44.810
 172     1.5     6     7    21   462        44.810
 173     1.5     6     7    23   552        44.810
 174     1.5     6     7    25   650        44.810
 175     1.5     6     7    27   756        44.810
 176     1.5     7     7    13   182        44.810
 177     1.5     7     7    15   240        44.810
 178     1.5     7     7    17   306        44.810
 179     1.5     7     7    19   380        44.810
 180     1.5     7     7    21   462        44.810
 181     1.5     7     7    23   552        44.810
 182     1.5     7     7    25   650        44.810
 183     1.5     7     7    27   756        44.810
 184     1.5     8     7    13   182        44.810
 185     1.5     8     7    15   240        44.810
 186     1.5     8     7    17   306        44.810
 187     1.5     8     7    19   380        44.810
 188     1.5     8     7    21   462        44.810
 189     1.5     8     7    23   552        44.810
 190     1.5     8     7    25   650        44.810
 191     1.5     8     7    27   756        44.810
 192     1.5     8     7    29   870        44.810
 193     0.5     7     8    13   182        57.580
 194     0.5     7     8    15   240        57.580
 195     0.5     7     8    17   306        57.580
 196     0.5     7     8    19   380        57.580
 197     0.5     7     8    21   462        57.580
 198     0.5     7     8    23   552        57.580
 199     0.5     7     8    25   650        57.580
 200     0.5     7     8    27   756        57.580
 201     0.5     8     8    13   182        57.580
 202     0.5     8     8    15   240        57.580
 203     0.5     8     8    17   306        57.580
 204     0.5     8     8    19   380        57.580
 205     0.5     8     8    21   462        57.580
 206     0.5     8     8    23   552        57.580
 207     0.5     8     8    25   650        57.580
 208     0.5     8     8    27   756        57.580
 209     0.5     8     8    29   870        57.580
 210     1.5     6     8    15   240        57.610
 211     1.5     6     8    17   306        57.610
 212     1.5     6     8    19   380        57.610
 213     1.5     6     8    21   462        57.610
 214     1.5     6     8    23   552        57.610
 215     1.5     6     8    25   650        57.610
 216     1.5     6     8    27   756        57.610
 217     1.5     7     8    13   182        57.610
 218     1.5     7     8    15   240        57.610
 219     1.5     7     8    17   306        57.610
 220     1.5     7     8    19   380        57.610
 221     1.5     7     8    21   462        57.610
 222     1.5     7     8    23   552        57.610
 223     1.5     7     8    25   650        57.610
 224     1.5     7     8    27   756        57.610
 225     1.5     8     8    13   182        57.610
 226     1.5     8     8    15   240        57.610
 227     1.5     8     8    17   306        57.610
 228     1.5     8     8    19   380        57.610
 229     1.5     8     8    21   462        57.610
 230     1.5     8     8    23   552        57.610
 231     1.5     8     8    25   650        57.610
 232     1.5     8     8    27   756        57.610
 233     1.5     8     8    29   870        57.610
 234     1.5     9     8    11   132        57.610
 235     1.5     9     8    13   182        57.610
 236     1.5     9     8    15   240        57.610
 237     1.5     9     8    17   306        57.610
 238     1.5     9     8    19   380        57.610
 239     1.5     9     8    21   462        57.610
 240     1.5     9     8    23   552        57.610
 241     1.5     9     8    25   650        57.610
 242     1.5     9     8    27   756        57.610
 243     1.5     9     8    29   870        57.610
 244     0.5     8     9    13   182        71.980
 245     0.5     8     9    15   240        71.980
 246     0.5     8     9    17   306        71.980
 247     0.5     8     9    19   380        71.980
 248     0.5     8     9    21   462        71.980
 249     0.5     8     9    23   552        71.980
 250     0.5     8     9    25   650        71.980
 251     0.5     8     9    27   756        71.980
 252     0.5     8     9    29   870        71.980
 253     0.5     9     9    11   132        71.980
 254     0.5     9     9    13   182        71.980
 255     0.5     9     9    15   240        71.980
 256     0.5     9     9    17   306        71.980
 257     0.5     9     9    19   380        71.980
 258     0.5     9     9    21   462        71.980
 259     0.5     9     9    23   552        71.980
 260     0.5     9     9    25   650        71.980
 261     0.5     9     9    27   756        71.980
 262     0.5     9     9    29   870        71.980
 263     1.5     7     9    13   182        72.010
 264     1.5     7     9    15   240        72.010
 265     1.5     7     9    17   306        72.010
 266     1.5     7     9    19   380        72.010
 267     1.5     7     9    21   462        72.010
 268     1.5     7     9    23   552        72.010
 269     1.5     7     9    25   650        72.010
 270     1.5     7     9    27   756        72.010
 271     1.5     8     9    13   182        72.010
 272     1.5     8     9    15   240        72.010
 273     1.5     8     9    17   306        72.010
 274     1.5     8     9    19   380        72.010
 275     1.5     8     9    21   462        72.010
 276     1.5     8     9    23   552        72.010
 277     1.5     8     9    25   650        72.010
 278     1.5     8     9    27   756        72.010
 279     1.5     8     9    29   870        72.010
 280     1.5     9     9    11   132        72.010
 281     1.5     9     9    13   182        72.010
 282     1.5     9     9    15   240        72.010
 283     1.5     9     9    17   306        72.010
 284     1.5     9     9    19   380        72.010
 285     1.5     9     9    21   462        72.010
 286     1.5     9     9    23   552        72.010
 287     1.5     9     9    25   650        72.010
 288     1.5     9     9    27   756        72.010
 289     1.5     9     9    29   870        72.010
 290     1.5    10     9    11   132        72.010
 291     1.5    10     9    13   182        72.010
 292     1.5    10     9    15   240        72.010
 293     1.5    10     9    17   306        72.010
 294     1.5    10     9    19   380        72.010
 295     1.5    10     9    21   462        72.010
 296     1.5    10     9    23   552        72.010
 297     1.5    10     9    25   650        72.010
 298     1.5    10     9    27   756        72.010
 299     1.5    10     9    29   870        72.010
 300     1.5    10     9    31   992        72.010
 301     0.5     9    10    11   132        87.980
 302     0.5     9    10    13   182        87.980
 303     0.5     9    10    15   240        87.980
 304     0.5     9    10    17   306        87.980
 305     0.5     9    10    19   380        87.980
 306     0.5     9    10    21   462        87.980
 307     0.5     9    10    23   552        87.980
 308     0.5     9    10    25   650        87.980
 309     0.5     9    10    27   756        87.980
 310     0.5     9    10    29   870        87.980
 311     0.5    10    10    11   132        87.980
 312     0.5    10    10    13   182        87.980
 313     0.5    10    10    15   240        87.980
 314     0.5    10    10    17   306        87.980
 315     0.5    10    10    19   380        87.980
 316     0.5    10    10    21   462        87.980
 317     0.5    10    10    23   552        87.980
 318     0.5    10    10    25   650        87.980
 319     0.5    10    10    27   756        87.980
 320     0.5    10    10    29   870        87.980
 321     0.5    10    10    31   992        87.980
 322     1.5     8    10    13   182        88.010
 323     1.5     8    10    15   240        88.010
 324     1.5     8    10    17   306        88.010
 325     1.5     8    10    19   380        88.010
 326     1.5     8    10    21   462        88.010
 327     1.5     8    10    23   552        88.010
 328     1.5     8    10    25   650        88.010
 329     1.5     8    10    27   756        88.010
 330     1.5     8    10    29   870        88.010
 331     1.5     9    10    11   132        88.010
 332     1.5     9    10    13   182        88.010
 333     1.5     9    10    15   240        88.010
 334     1.5     9    10    17   306        88.010
 335     1.5     9    10    19   380        88.010
 336     1.5     9    10    21   462        88.010
 337     1.5     9    10    23   552        88.010
 338     1.5     9    10    25   650        88.010
 339     1.5     9    10    27   756        88.010
 340     1.5     9    10    29   870        88.010
 341     1.5    10    10    11   132        88.010
 342     1.5    10    10    13   182        88.010
 343     1.5    10    10    15   240        88.010
 344     1.5    10    10    17   306        88.010
 345     1.5    10    10    19   380        88.010
 346     1.5    10    10    21   462        88.010
 347     1.5    10    10    23   552        88.010
 348     1.5    10    10    25   650        88.010
 349     1.5    10    10    27   756        88.010
 350     1.5    10    10    29   870        88.010
 351     1.5    10    10    31   992        88.010
 352     1.5    11    10     9    90        88.010
 353     1.5    11    10    11   132        88.010
 354     1.5    11    10    13   182        88.010
 355     1.5    11    10    15   240        88.010
 356     1.5    11    10    17   306        88.010
 357     1.5    11    10    19   380        88.010
 358     1.5    11    10    21   462        88.010
 359     1.5    11    10    23   552        88.010
 360     1.5    11    10    25   650        88.010
 361     1.5    11    10    27   756        88.010
 362     1.5    11    10    29   870        88.010
 363     1.5    11    10    31   992        88.010
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS 50736
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO  113.3 %
      CURRENT DATE:  Jan 24 11:02:02 2023
 ===============================================================================
